### PR TITLE
Adds translations for gist component in service config

### LINKED ISSUE #9280: Add Translations for Gist component strong element
**Is your feature request related to a problem? Please describe.**
We need to support all our services with localisation for the gist component.

**Describe the solution you'd like**
Add translations for each service to the service config for the gist component integrated in a previous issue.
Sport, Cymru Fyw, Naidheachdan and Newsround will not support the Gist so can we default to the English "At a glance" 

Service | Text
-- | --
News | At a glance
Sport | At a glance
Naidheachdan | At a glance
Newsround | At a glance
Cymru Fyw | At a glance
Persian | خلاصه
Igbo | Nchịkọta
Pidgin | Summary
Yoruba | Ìsọníṣókí
Urdu | خلاصہ
Pashto | لنډیز
Arabic | ملخص
Thai | สรุป
Punjabi | ਸਾਰ
Afaan Oromoo | Guduunfaa
Amharic | ጭምቅ ሃሳብ
Indonesian | Rangkuman
Tigrinya |  
Korean | 요약
Afrique | Points clés
Gahuza | Incamake
Somali | Kooban
Swahili | Muhtasari
Azeri | Xülasə
Hausa | Takaitacce
Vietnamese | Tóm tắt
Kyrgyz | Корутунду
Japanese | 要点
Nepali | सारांश
Uzbek (Cyrillic) | Хулоса
Sinhala | සාරාංශය
Bengali | সার সংক্ষেপ
UK China (Simp) | 概要
UK China (Trad) | 概要
Turkce | Özet
Burmese (Unicode) | အနှစ်ချုပ်
Gujarati | સારાંશ
Marathi | थोडक्यात
BBC News Mundo | Sumario
Portuguese | Pontos-chave
Russian | Коротко
Serbian(Cyr) | Сажетак
Serbian (Lat) | Sažetak
Tamil | முக்கிய சாராம்சம்
Telugu | సారాంశం
Ukrainian | Стисло
Hindi | सारांश
Zhongwen (Simp) | 概要
Zhongwen (Trad) | 概要

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves #9280 

**Overall change:**
Adds gist component translations to service config files.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/azeri.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ export const service = {
         audioPlayer: 'Audio player',
         videoPlayer: 'Video player',
       },
+      gist: 'Xülasə',
```
Comment (ryanmccombe) [inline src/app/lib/config/services/azeri.js:77]: Got a rectangle of doom here - encoding issue? Xülasə

Comment (HarveyPeachey) [reply src/app/lib/config/services/azeri.js:77]: @ryanmccombe Hmm I can't see the Doomtangle, which character position wise is showing that?

File: src/app/lib/config/services/afrique.js [hunk 2]
```diff
@@ -75,6 +75,7 @@ export const service = {
         audioPlayer: 'Lecteur audio',
         videoPlayer: 'Lecteur vidéo',
       },
+      gist: '	Points clés',
```
Comment (HarveyPeachey) [inline src/app/lib/config/services/afrique.js]: Just to double check, some of the translations have a space at the beginning, is this intended? 

I've seen it on translations for these services:
Afrique
Amharic 
Somali
Tamil
Turkce
Ukchina

Comment (pvaliani) [reply src/app/lib/config/services/afrique.js]: Will fix and push, likely a copying error. Thank you

File: src/app/lib/config/services/azeri.js (hunk 1 above)
Comment (pvaliani) [reply src/app/lib/config/services/azeri.js:77]: Can someone clarify the Doomtangle? Sounds intriguing..

File: src/app/lib/config/services/afrique.js (hunk 2 above)
Comment (pvaliani) [reply src/app/lib/config/services/afrique.js]: @HarveyPeachey should be fixed in latest commit - good spot! Apologies

File: src/app/lib/config/services/azeri.js (hunk 1 above)
Comment (ryanmccombe) [reply src/app/lib/config/services/azeri.js:77]: Oh interesting - only an issue on mobile chrome maybe?

Screenshot_20210809-134004

File: src/app/lib/config/services/afrique.js (hunk 2 above)
Comment (HarveyPeachey) [reply src/app/lib/config/services/afrique.js]: @pvaliani No worries, it's fine. It's happened before when I've added translations, it's a common thing don't worry

File: src/app/lib/config/services/azeri.js (hunk 1 above)
Comment (HarveyPeachey) [reply src/app/lib/config/services/azeri.js:77]: @ryanmccombe That's weird, does it show the rectangle when you view the file through the files changed tab?

Comment (ryanmccombe) [reply src/app/lib/config/services/azeri.js:77]: Yep - looking closer, this file has numerous rectangles on mobile though so probably just a GitHub or android bug - not a simorgh issue

**Review by ryanmccombe — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]

**Review by rebeccamcginn — APPROVED** [review]

**Pull request merged**